builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -848,8 +848,8 @@ def cleanupS3BucketPolicy(stage, assetInfo) {
  */
 def resetCredentials(credsId) {
 	echo "resetting AWS credentials"
-	sh "aws configure set profile.${credsId}.aws_access_key_id XXXXXXXXXXXXXXXXXXXXXXXXXX"
-	sh "aws configure set profile.${credsId}.aws_secret_access_key XXXXXXXXXXXXXXXXXXXXXX"
+  def credPath = System.getenv().HOME + "/.aws/credentials"
+	sh "sed -i '/${credsId}/,+2d' ${credPath}"
```

Comment:
@rajeevr2715  Can you check if we'll need to do something similar in  ~/.aws/config?

Reply:
Yes. removed that also.